### DISCUSSION TITLE
Welcome to .NET nanoFramework Discussions!

### DISCUSSION THREAD

**Discussion by josesimoes** [nanoframework/Home#695] [General]
## 👋 Welcome!
  We’re using Discussions as an alternative place to ask questions. [Discord](https://discordapp.com/invite/gCyBu8T) is still the preferred place for conversations and community interaction and there is also [Stack Overflow](https://stackoverflow.com/questions/tagged/nanoframework). This one here is another channel for people that wont/can't use the above.
  * Before posting a question please use the search box to find if the same (similar question) has been asked before.
  * We want to keep this place happy and productive. No spamming, no NSFW, don't be rude, don't be annoying.
  * We are happy to help where we can, but come with at least a basic understanding of C#. Review the samples in the samples repo or do a smaller project first. Some of the class libs have APIs and techniques that can be not so obvious for beginners.
  * When bug-fixing, attempt to solve problems on your own first. If you get stuck, let us know the things you tried. We won't build your app for you.

**Comment by MichalMayerCz** [comment]
hi, i tried log to discord, but the invitation link to discord seems be invalid. can you check it ? thank you

**Reply by josesimoes** [reply]
Hi @MichalMayerCz 

This is the one you've used? https://discord.gg/gCyBu8T

**Reply by MichalMayerCz** [reply]
yes, but doesnt work (message is localized to my language sorry, i cant switch it)
![image](https://user-images.githubusercontent.com/80054394/110004055-4d53f600-7d17-11eb-89e3-73f8daa6d12f.png)

**Reply by MichalMayerCz** [reply]
i am sorry, when i have it tried in edge, it works. before i did it in chrome... i dont understand why...

**Reply by shawty** [reply]
Yes I got much the same too... that's why I came here.  In edge (The ONLY browser I have installed) and "something went wrong, we have to verify you by phone, please give us your phone number..."

Erm... let me see.... NO, you don't NEED my phone number to do this...

Oh and what's with the needing to "confirm my age" before I'm allowed to log out too!!!

All sly/snide attempts to grab personal data off of me, thats what they are

**Reply by shawty** [reply]
anyway... sorry about that... rant is over now...

My ESP32 has stopped deploying, and I've been able to sort it out, where can I ask for a hand?

**Reply by josesimoes** [reply]
> anyway... sorry about that... rant is over now...
> 
> My ESP32 has stopped deploying, and I've been able to sort it out, where can I ask for a hand?
> 

Here's the link to Discord https://discordapp.com/invite/gCyBu8T

That's the best place to keep a conversation and get quick help.

BTW, any personal data that you're request from Discord its up to them we Do NOT ask/have access to any of those. 😉

**Reply by shawty** [reply]
Hi Jose, ya I get that...   I was just having a rant at them.. as you know I got in, and we got things sorted.. thanks again.
